### PR TITLE
Schema varchar

### PR DESCRIPTION
**Pull request by sopel39**

### CODE REVIEW

File: tempto-core/src/main/java/com/teradata/tempto/internal/fulfillment/table/hive/HiveTableManager.java [hunk 1]
```diff
@@ -50,19 +50,22 @@
...
+    private final String hiveDatabasePath;
 
     @Inject
     public HiveTableManager(QueryExecutor queryExecutor,
             HdfsDataSourceWriter hdfsDataSourceWriter,
             TableNameGenerator tableNameGenerator,
             @Named("tests.hdfs.path") String testDataBasePath,
-            @Named("databaseName") String databaseName)
+            @Named("databaseName") String databaseName,
+            @Named("hive.database.path") String hiveDatabasePath)
```
Comment (losipiuk) [inline tempto-core/src/main/java/com/teradata/tempto/internal/fulfillment/table/hive/HiveTableManager.java]: drop `hive` prefix. At least from java parameter name.

File: tempto-examples/src/main/resources/tempto-configuration.yaml [hunk 2]
```diff
@@ -57,6 +57,10 @@ tests:
   hdfs:
     path: /tempto
 
+hive:
+  databse:
```
Comment (losipiuk) [inline tempto-examples/src/main/resources/tempto-configuration.yaml]: typo

**Comment by losipiuk** [comment]
looks goofd

**Comment by losipiuk** [comment]
d

File: tempto-core/src/main/java/com/teradata/tempto/fulfillment/table/hive/tpch/TpchTableDefinitions.java [hunk 3]
```diff
@@ -25,9 +28,9 @@
                     .setCreateTableDDLTemplate("" +
                             "CREATE %EXTERNAL% TABLE %NAME%(" +
                             "   n_nationkey     BIGINT," +
-                            "   n_name          STRING," +
+                            "   n_name          VARCHAR(25)," +
```
Comment (sopel39) [inline tempto-core/src/main/java/com/teradata/tempto/fulfillment/table/hive/tpch/TpchTableDefinitions.java]: yes

**Pull request closed without merging**